src/keycloak.js

```diff
@@ -0,0 +1,12 @@
+import Keycloak from "keycloak-js";
+
+// Setup Keycloak instance as needed
+// Pass initialization options as required  TODO
+
+const keycloak = new Keycloak({
```

Comment:
Please do not hardcode things. Better to add to the variables file and add defaults for development perhaps

Reply:
next pr

Reply:
fix it in this one please as part of this review